src/main/java/seedu/address/model/StatisticsCalculator.java

```diff
@@ -33,24 +33,32 @@ public int getSize() {
     /**
      * Sums up the total money owed by the people in AddressBook.
```

Comment:
Seems like getAmountOwed() and getAmountPaid() comments are missing the return tag?

Reply:
Oops missed out, have updated it!